webapp/src/css/inbox.less

```diff
@@ -1304,20 +1219,67 @@ mm-search-bar {
 }
 
 @media (max-width: @media-mobile) {
-  .old-nav {
-    .content {
-      .page {
-        top: 115px;
-      }
-    }
-  }
-
   .fast-action-trigger:not(.embed-fast-action) {
     position: fixed;
-    bottom: 17px;
+    bottom: calc(@bottom-navbar-height + @fab-bottom-padding);
     right: 18px;
     top: auto;
   }
+  .header {
+    .tabs {
+      display: flex;
+      flex-direction: row;
+      padding: 0 15px;
+      a {
+        flex-grow: 1;
+        min-width: 70px;
+        max-width: 100px;
+      }
+    }
+  }
+  .app-root:not(.old-nav) {
+    .header {
+      display: block;
+      width: 100vw;
+      background-color: @bottom-navbar-color;
+      position: fixed;
+      bottom: 0;
+
+      .inner {
+        height: @bottom-navbar-height;
+      }
+
+      .logo,
+      .extras {
+        display: none;
+      }
+
+      .tabs {
+        .button-label {
+          top: 0;
+          font-size: @font-extra-smaller;
+          padding-top: 5px;
+        }
+        a {
+          min-width: 0;
+          .mm-icon {
+            padding: 8px 0;
+          }
+          &.selected
+          .mm-icon {
+            border-radius: 999px;
```

Comment:
@Benmuiruri, can you show me the component on the page where this 999px is applying? I can't seem to find it
It's kinda interesting to see a big radius like that

Reply:
That's an old CSS shortcut to create pill shaped border radius.